### PR TITLE
Update tests readme with tdqubit formulas

### PR DESCRIPTION
**Pull request by gautierronan**
Closes https://linear.app/dynamiqs/issue/DYN-85.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -8,42 +8,44 @@ We test our solvers on systems with an analytical solution.
...
+- Cost function: $C(\rho) = \mathrm{Tr}[a^\dagger a \rho]$
 - Gradient parameters: $\theta=(\Delta, \alpha_0, \kappa)$
 
 **Solution at time $t$**
 
 - State:
 
 $$
-\ket{\psi(t)} = \ket{\alpha(t)}\ \text{with}\ \alpha(t) = \alpha_0 e^{-i\Delta t - \kappa t/2}
+\rho_t = \vert{\alpha(t)}\rangle\!\langle{\alpha(t)}\vert \ \text{with} \ \alpha(t) = \alpha_0 e^{-i\Delta t - \kappa t/2}
```
Comment (pierreguilmin) [inline tests/README.md:23]:
```suggestion
\rho_t = \vert{\alpha(t)}\rangle\\langle{\alpha(t)}\vert \ \text{with} \ \alpha(t) = \alpha_0 e^{-i\Delta t - \kappa t/2}
```

Comment (pierreguilmin) [reply tests/README.md:23]: Why not using `\bra` and `\ket`?

**Review by pierreguilmin — DISMISSED** [review]
Nice 🙏

**Comment by pierreguilmin** [comment]
Do you have a proof for the formulas or did you copy-paste the working formulas we had?

**Comment by gautierronan** [comment]
> Do you have a proof for the formulas or did you copy-paste the working formulas we had?

Well I did a pen and paper derivation, it's quite straightforward TBH.

**Comment by pierreguilmin** [comment]
The `!` doesn't render on GitHub 😉 
Screenshot 2024-02-09 at 14 59 00

**Review by pierreguilmin — APPROVED** [review]
Thanks a lot!

**Pull request merged**